live-examples/html-examples/global-attributes/css/accesskey.css

```diff
@@ -0,0 +1,12 @@
+@font-face {
```

Comment:
I just noticed this. I think the other pull requests also had it. You "should" not need to do this as we are adding fonts here:

https://github.com/mdn/interactive-examples/blob/master/css/editor-libs/shadow-fonts.css

There was an issue where custom fonts would not work inside the showDOM output, and after some research I found info that required the above change. Does it not work for you without the `@font-face` block? If not, does it not work in all browsers or only a specific one?

Reply:
I just added it because I thought it was still needed. I'll not do that again then. ;-)